src/trix/controllers/level_2_input_controller.js

```diff
@@ -136,7 +137,7 @@ export default class Level2InputController extends InputController {
     compositionend(event) {
       if (this.composing) {
         this.composing = false
-        return this.scheduleRender()
+        if (!config.browser.composesOnCursorMove) this.scheduleRender()
```

Comment:
We discussed a bit in the campfire already, but my questions around this are really:

1. Do we _ever_ need to render here? It looks a bit like we don't, but it's hard to be sure. There is a failing test if you take it out, but the test could also be misleading.
2. If we can't take it out all the time, then instead of detecting the device, could we do something like remember the value of the last `input` event, and only render if we get a `compositionend` that has a different value? That might be a way to tell a "problematic `compositionevent`" from an expected one?